cdc/owner/gc_manager.go

```diff
@@ -37,6 +37,13 @@ const (
 // gcSafepointUpdateInterval is the minimum interval that CDC can update gc safepoint
 var gcSafepointUpdateInterval = 1 * time.Minute
 
+// GcManager is an interface for gc manager
+type GcManager interface {
+	updateGCSafePoint(ctx cdcContext.Context, state *model.GlobalReactorState) error
+	currentTimeFromPDCached(ctx cdcContext.Context) (time.Time, error)
+	CheckStaleCheckpointTs(ctx cdcContext.Context, checkpointTs model.Ts) error
```

Comment:
Why is this function exported but the other two are not

Reply:
after analyzing the code, no need to export `CheckStaleCheckpointTs` method, make it unexported.